src/main/java/org/mastodon/mamut/tomancak/lineage_registration/RootsPairing.java

```diff
@@ -0,0 +1,61 @@
+package org.mastodon.mamut.tomancak.lineage_registration;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.mastodon.collection.ObjectRefMap;
+import org.mastodon.collection.RefRefMap;
+import org.mastodon.collection.RefSet;
+import org.mastodon.collection.ref.ObjectRefHashMap;
+import org.mastodon.collection.ref.RefRefHashMap;
+import org.mastodon.mamut.model.ModelGraph;
+import org.mastodon.mamut.model.Spot;
+
+/**
+ * Pair the root spots in two ModelGraphs based on their label.
+ */
+public class RootsPairing
```

Comment:
It would be nice, if this class would also be available to other plugin repositories.

Reply:
Please include this in the issue mentioned above.